wordpress-mobile/WordPress-iOS
# Add "Jetpack powered" badge to My Site Home

Fixes #NA

This PR adds a "Jetpack Powered" badge at the bottom of the Home Screen in My Site

iPad Dark | iPhone Light
-- | --
<img height=400 src=https://user-images.githubusercontent.com/34376330/180483439-ef08d157-86bd-4d88-9696-3353d0b4734c.png> | <img height=350 src="https://user-images.githubusercontent.com/34376330/180484064-6ec43fd9-28a1-4cee-86ef-11cb0672e385.png">



To test:

- Build/run the WordPress target on this branch
- Go to My Site then select Home
- Make sure the Jetpack badge appears at the bottom of the list
- Stop execution and run the Jetpack target and repeat the above steps
- Make sure the badge does not appear

## Regression Notes
1. Potential unintended areas of impact
My Site Home (very low)

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test on a few devices

3. What automated tests I added (or what prevented me from doing so)
none

PR submission checklist:

- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ Internal RNs will be added in a separate PR
